base/applications/msconfig/autochkpage.c

```diff
@@ -0,0 +1,247 @@
+/*
+ * PROJECT:     ReactOS Applications
+ * LICENSE:     LGPL - See COPYING in the top level directory
+ * PURPOSE:     Auto Check (autochk) page handler
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "precomp.h"
+#include "resource.h"
+#include "autochkpage.h"
+
+#include <tchar.h>
+#include <windows.h>
+
+/* VARIABLES *******************************************************************/
+
+HWND hAutoChkPage;
+HWND hAutoChkDialog;
+HKEY hKey;
+LONG Status;
+TCHAR szValueData[255];
+DWORD dwcbData = sizeof(szValueData);
+DWORD dwBufferSize = 0;
+LPWSTR lpsszMultiString = NULL;
+size_t szSize;
+LPWSTR ptr;
+DWORD dwType;
+TCHAR Program[] = _T("System32\\autochk.exe");
+TCHAR SubKey[] = _T("SYSTEM\\CurrentControlSet\\Control\\Session Manager");
+TCHAR ValueName[] = _T("BootExecute");
+TCHAR EnableValue[] = _T("autocheck autochk *\0");
+
+/* FUNCTIONS *******************************************************************/
+
+INT_PTR CALLBACK
+AutoChkPageWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
+{
+  UNREFERENCED_PARAMETER(lParam);
+  
+  switch (message)
+  {
+    case WM_INITDIALOG:
+    {
+      hAutoChkDialog = hDlg;
+      SetWindowPos(hDlg, NULL, 10, 32, 0, 0, SWP_NOACTIVATE | SWP_NOOWNERZORDER | SWP_NOSIZE | SWP_NOZORDER);
+
+      // Open the key so we can query it
+      Status = RegOpenKeyEx(HKEY_LOCAL_MACHINE,
```

Comment:
Please avoid `Status` for non-NTSTATUS variables. I would suggest `Error` but there are many other options of course.